### PR TITLE
Fix experimental badge on new Dev UI

### LINKED ISSUE #33009: new devui does not show experimental status
### Describe the bug

old:

new:

### Expected behavior

that experimental still shown

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by marcozanghi**
Fix #33009

### CODE REVIEW

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js [hunk 1]
```diff
@@ -101,21 +121,31 @@ export class QwcExtensions extends observeState(LitElement) {
...
     }
 
     _renderDefaultCardContent(extension){
-        
-        return html`<div class="card-content" slot="content">
+        if(extension.status === "experimental") {
+            return html`
+                <div class="card-content" slot="content">
+                    <span class="description">
+                        <span class="float-right badge badge-warning">EXPERIMENTAL</span>
```
Comment (phillip-kruger) [inline extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js:134]: You should use `qui-badge`. See https://github.com/quarkusio/quarkus/blob/main/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js

Here is an example where it's used:

https://github.com/quarkusio/quarkus/blob/b2f79310c2eb202a184838aa2fa4d558f03f9ecb/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-server-log.js#L12

and 

https://github.com/quarkusio/quarkus/blob/b2f79310c2eb202a184838aa2fa4d558f03f9ecb/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-server-log.js#L265

You would add `theme="warning"`

Thanks !

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js [hunk 2]
```diff
@@ -153,10 +183,35 @@ export class QwcExtensions extends observeState(LitElement) {
                     providesCapabilities="${extension.providesCapabilities}"
                     extensionDependencies="${extension.extensionDependencies}">
                     <div class="card-content" slot="content">
+                        <span class="float-right badge badge-warning">EXPERIMENTAL</span>
```
Comment (phillip-kruger) [inline extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js:186]: You can probably remove this duplication (in the else) and just here call a method that render the badge if needed

**Review by phillip-kruger — CHANGES_REQUESTED** [review]

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js [hunk 3]
```diff
@@ -80,6 +80,7 @@ export class QuiBadge extends LitElement {
         icon: {type: String},
         level: {type: String},
         small: {type: Boolean},
+        warning: {type: Boolean},
```
Comment (phillip-kruger) [inline extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js:83]: You do not need to add this. You can just use it as is: see this for a warning example: https://github.com/phillip-kruger/quarkus-jokes/blob/63db547df6f0b548cb1cebf47062da23ef4b7b95/deployment/src/main/resources/dev-ui/qwc-jokes-vaadin.js#LL222C29-L222C29

You can remove all your modification in qui-badge

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js [hunk 4]
```diff
@@ -135,28 +143,33 @@ export class QwcExtensions extends observeState(LitElement) {
...
+                    ${this._renderExperimentalBadge(extension)}
+                    ${extension.description}
+                </div>    
+            </qwc-extension>`;
+        }
+    }
+
+    _renderExperimentalBadge(extension){
+        if(extension.status === "experimental") {
+            return html`<qui-badge small warning class="float-right"><span>EXPERIMENTAL</span></qui-badge>`;
```
Comment (phillip-kruger) [inline extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js:172]:
```suggestion
            return html`<qui-badge level="warning" class="float-right" small><span>EXPERIMENTAL</span></qui-badge>`;
```

**Review by phillip-kruger — CHANGES_REQUESTED** [review]

**Comment by phillip-kruger** [comment]
@marcozanghi Also please squash into one commit :)

Thanks !

File: extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js (hunk 3 above)
Comment (marcozanghi) [reply extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js:83]: @phillip-kruger  ok ! If you agree I will delete only line 98 where "this.small = false;" is duplicated.

**Review by phillip-kruger — APPROVED** [review]
Thanks !

**Comment by phillip-kruger** [comment]
@gsmet can you confirm that we can merge here. We have CI failures that seems unrelated

**Comment by geoand** [comment]
Can you please rebase onto `main`?

**Pull request merged**